docs/contributing/Setting-up-the-Engine-development-environment.md

```diff
@@ -26,7 +26,7 @@ Make sure you have the following dependencies available:
  * On macOS:
    - Install the latest Xcode.
    - On Apple Silicon arm64 Macs, install the Rosetta translation environment by running `softwareupdate --install-rosetta`.
-   - Install Oracle's Java JDK, version 1.8 or later.
+   - Add Java to path
```

Comment:
I think the real fix is to remove this line and fix the formatter to not require it, since nothing else about the engine requires `java` on your path other than the pre-commit formatter?

I'm happy to send a change to the formatter to have it depend on engine/src/flutter/third_party/java/openjdk/Contents/Home/bin/java if you'd be OK with that.

Reply:
I am ok with that.